### PR TITLE
2415 > 2416

### PR DESCRIPTION
**Pull request by XSlayer300**
## About The Pull Request
Per title.

## Why It's Good For The Game
Immersion.

## Changelog
:cl:
tweak: The in-game year is now 2416.
tweak: In addition to this, the ingame year is now dynamic.
/:cl:

### CODE REVIEW

**Comment by Tennessee116** [comment]
How about, instead of doing this manually ever year, we implement something like this (with our own timeline):

`var/game_year           = (text2num(time2text(world.realtime, "YYYY")) + 288)`

Source: [Bay config file](https://github.com/Baystation12/Baystation12/blob/6c886a572fe6f7a8a8ae339496772a7f802ddb98/code/_global_vars/configuration.dm)

**Comment by BraveMole** [comment]
I can't play without this, please speed merge

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -319,7 +319,7 @@
 	output += "<a href='byond://?src=[REF(src)];lobby_choice=marines'>TerraGov Marine Corps</A><br><br><a href='byond://?src=[REF(src)];lobby_choice=aliens'>Xenomorph Hive</A>"
 	output += "</div>"
 
-	var/datum/browser/popup = new(src, "lore", "<div align='center'>Current Year: 2415</div>", 240, 300)
+	var/datum/browser/popup = new(src, "lore", "<div align='center'>Current Year: 2416</div>", 240, 300)
```
Comment (TiviPlus) [inline code/modules/mob/new_player/new_player.dm:322]: ake this use the define please

Comment (XSlayer300) [reply code/modules/mob/new_player/new_player.dm]: done

**Comment by Tennessee116** [comment]
I'd advise changing the changelog to something like,

> The in-game year is no longer static.

rather than just saying the in-game year is 2416.

**Comment by XSlayer300** [comment]
> I'd advise changing the changelog to something like,
> 
> > The in-game year is no longer static.
> 
> rather than just saying the in-game year is 2416.

Done.

**Review by TiviPlus — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-1). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**